installer/inventory

```diff
@@ -152,3 +152,7 @@ secret_key=awxsecret
 # which makes include "optional" - i.e. not fail
 # if file is absent
 #extra_nginx_include="/etc/nginx/awx_extra[.]conf"
+#
+# Allow for different docker logging drivers
+# See: https://docs.docker.com/config/containers/logging/configure/
+# docker_logger=journald
```

Comment:
As far as I know, `journald` tends to rate-limit the logging input so that some logs can go missing.

How about instead:
- keeping `json-file` as the default logging option ?
- providing instructions to leverage `journald` docker logging and changing the rate-limiting paramaters of `journald` ?

Reply:
I did not change the default logger; I only allowed for using journald if desired.  I can add a comment about journald and rate-limiting if you'd like.

Reply:
Added more comments.